pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/alder/EndringAvUttaksgradAuto.kt

```diff
@@ -0,0 +1,250 @@
+                                bokmal { +"Du får fortsatt utbetalt skjermingstillegget til uføre. Vedtaket er gjort etter folketrygdloven §§ 19-9a, 19-10 og 19-12." },
+                                nynorsk { +"Du får fortsatt utbetalt skjermingstillegget til uføre. Vedtaket er gjort etter folketrygdlova §§ 19-9a, 19-10 og 19-12." },
+                                english { +"You will still receive the supplement for disabled people. This decision was made pursuant to the provisions of §§ 19-9a, 19-10 and 19-12 of the National Insurance Act." }
+                            )
+                        }
+                    }
+                }
+
+                showIf(regelverkType.isOneOf(AP2011)) {
+                    // endrUtaksgradAP2011
+                    paragraph {
+                        text(
+                            bokmal { +"Vedtaket er gjort etter folketrygdloven §§ 19-10, 19-12 og 22-12." },
+                            nynorsk { +"Vedtaket er gjort etter folketrygdlova §§ 19-10, 19-12 og 22-12." },
+                            english { +"This decision was made pursuant to the provisions of §§ 19-10, 19-12 and 22-12 of the National Insurance Act." }
+                        )
+                    }
+                }.orShowIf(regelverkType.isOneOf(AP2016)) {
+                    // endrUtaksgradAP2016
+                    paragraph {
+                        text(
+                            bokmal { +"Vedtaket er gjort etter folketrygdloven §§ 19-10, 19-12, 19-15, 20-14, 20-16, 20-19 og 22-12." },
+                            nynorsk { +"Vedtaket er gjort etter folketrygdlova §§ 19-10, 19-12, 19-15, 20-14, 20-16, 20-19 og 22-12." },
+                            english { +"This decision was made pursuant to the provisions of §§ 19-10, 19-12, 19-15, 20-14, 20-16, 20-19 and 22-12 of the National Insurance Act." }
+                        )
+                    }
+                }.orShowIf(regelverkType.isOneOf(AP2025)) {
+                    // endrUtaksgradAP2025Soknad
+                    paragraph {
+                        text(
+                            bokmal { +"Vedtaket er gjort etter folketrygdloven §§ 20-14, 20-16 og 22-13." },
+                            nynorsk { +"Vedtaket er gjort etter folketrygdlova §§ 20-14, 20-16 og 22-13." },
+                            english { +"This decision was made pursuant to the provisions of §§ 20-14, 20-16 and 22-13 of the National Insurance Act." }
+                        )
+                    }
+                }
+
+                showIf(alderspensjonVedVirk.uttaksgrad.lessThan(100)) {
+                    // gradsendrAPSoknadInfo, nySoknadAPInfo
+                    paragraph {
+                        text(
+                            bokmal {
+                                +"Du må sende oss en ny søknad når du ønsker å ta ut "
+                                +ifElse(
+                                    alderspensjonVedVirk.uttaksgrad.greaterThan(0),
```

Comment:
Trekk gjerne ut condition som variabel her (rett under paragraph, feks: val uttaksgradStoerreEnn0 = alderspensjonVedVirk.uttaksgrad.greaterThan(0)), og bruk denne i ifElse på alle tre språk.Ved å redeklarere condition for kvart språk er det risiko for at nokon seinare kåler det til og endrar utan å endre alle.

Reply:
Takk!